venues/EMNLP/python/init.py

```diff
@@ -23,6 +23,9 @@
 print('connecting to {0}'.format(client.baseurl))
 
 emnlp_request_id = 'rylllwkbKN'
+start_date = datetime.datetime(2019, 4, 2, 13, 30)
```

Comment:
you should read the dates from the request form note.

Reply:
When I went to make this change, I realized that I'd need to copy/paste a lot of the logic from the `get_conference` function. Instead I thought it would be better to make a small change to that function. 

See https://github.com/iesl/openreview-py/pull/280 and let me know what you think. I don't mind copy/pasting if you don't like this solution.